### DISCUSSION TITLE
Mech armor

### DISCUSSION THREAD

**Discussion by Rearth** [Rearth/Oritech#8] [Todos]
As an alternative to the light exosuit, a heavy mech armor would be cool. This grants massive defense bonus, but is heavier and slower than the exosuit or the normal player (movement is slowed down when wearing this). The intended use is for combat mainly here. Tech-tree wise it is positioned similar to the exosuit

**Comment by 0wdg** [comment]
would be really if there was like weapons embedded into the mech suit like slots for weapons, obviously a jetpack slot, cooler if there’s smooth animations, also a blade embedded or even a gunblade could be cool

**Comment by Gahedes** [comment]
When will our heavy mech armor be updated? I can't wait any longer.(●'◡'●)

**Comment by RubyCascade** [comment]
How 'heavy' exactly do you want the heavy mech armor to be? Because I have some ideas for how to sell the feeling of it being heavy. And I have some ideas for what it could do.

Do you want it to be more towards normal generic sci-fi heavy infantry or more towards space marine?
